boudicca.base/fetcher-lib/src/main/kotlin/base/boudicca/fetcher/Fetcher.kt

```diff
@@ -10,9 +10,13 @@ import java.net.http.HttpResponse.BodyHandlers
 import java.time.Clock
 import java.util.concurrent.Callable
 
+private const val WAIT_TIME_FACTOR = 1.5F
+private const val MIN_WAIT_TIME = 100L
```

Comment:
is the suggested case of parameters, enum etc. enforced by detekt? https://kotlinlang.org/docs/coding-conventions.html#property-names

Reply:
yes, at least for enums, pretty sure for constants as well